t/porting/libperl.t

```diff
@@ -45,6 +45,11 @@ use strict;
 
 use Config;
 
+# maint (and tarballs of maint releases) may not have updates here to
+# deal with changes to nm's output in some toolchains
+$^V =~ /^v\d\.\d*[13579]\./
```

Comment:
Technically, that first `\d` should be `\d+`.

Reply:
I agree with @mauke on the regex; otherwise LGTM.